src/components/submission/metadata-row.vue

```diff
@@ -131,6 +133,9 @@ export default {
         __system.attachmentsPresent !== __system.attachmentsExpected;
     },
     editPath() {
+      if (this.form.dataExits && this.form.webFormsEnabled) {
```

Comment:
I think `dataExits` needs to be `dataExists`. But I'm also thinking that we could consider moving the check for `form.dataExists` from `SubmissionMetadataRow` to the `v-if` in `SubmissionTable`. We already check `project.dataExists` there. Instead of checking `form.dataExists` in each row, we could wait to render the table until the form response has been received. I think either way works fine though.